### ISSUE TITLE
does it support multi-GPU training?

### ISSUE THREAD

**Issue by zhibo-liu** [junyanz/VON#6]
Hi, I found it that during trianing, mult GPU didn't work.
For example
```bash
bash ./scripts/train_shape.sh 0,1 car df 
```
only GPU 0 is working, the others only takes 10MB memory and 0% GPU Utility
```bash
| NVIDIA-SMI 390.67                 Driver Version: 390.67                    |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 108...  Off  | 00000000:05:00.0 Off |                  N/A |
| 49%   82C    P2    88W / 250W |   5849MiB / 11176MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GTX 108...  Off  | 00000000:06:00.0 Off |                  N/A |
| 26%   43C    P8     9W / 250W |     10MiB / 11178MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
```

Thank you.

**Comment by ztzhang** [comment]
Hi, currently we do not support multi-GPU training. We will keep you posted once this is available.

**Comment by junyanz** [comment]
The shape training code should support mutliGPU training now. You may want to use `--G_norm_3D inst3d` rather than `batch3d' as the default PyTorch batchnorm does not support multi-GPU sync. I am still looking at the texture module.

**Comment by junyanz** [comment]
I also enabled multi-GPUs training for the texture module. Again, you may want to use `--norm inst`. We haven't tested the quality of results with multi-GPUs training.

**Comment by zhibo-liu** [comment]
thank you so much!
